CRM/Custom/Form/CustomDataByType.php

```diff
@@ -27,31 +27,39 @@
 class CRM_Custom_Form_CustomDataByType extends CRM_Core_Form {
 
   /**
-   * Contact ID associated with the Custom Data
-   *
-   * @var int
+   * @var array
    */
-  public $_contactID = NULL;
+  private $groupTree;
+
+  /**
+   * @var array
+   */
+  private $groupCount;
```

Comment:
Hi @eileenmcnaughton 

Any reason why this field is set to private not protected/public?